package.json

```diff
@@ -34,6 +34,7 @@
     "react-scripts": "^4.0.3",
     "redux": "^4.0.5",
     "redux-thunk": "^2.3.0",
+    "styled-components": "^5.3.0",
```

Comment:
adding another library and way of writing components feels like we're over complicating the project for limited gains. If Ant design is missing something, I'd rather have us write out a custom reusable component. Right now we're writing CSS in 4 different ways (separate CSS files, in-line `style:{}`, extracted CSS to tops of files, and styled-components) , this would also introduce another way of writing React components and it feels like not the best solution to the problem. 

Open to hearing about thoughts on it though

Reply:
We also observed the multiple ways styles were being defined.
We will work towards consolidating to use Ant Design's LESS mechanism in PR #71.